Jenkinsfile

```diff
@@ -26,7 +26,7 @@ pipeline {
                        queue: 'osci-pipelines-queue-14'
                    ],
                    checks: [
-                       [field: '$.update.release.dist_tag', expectedValue: '^f[3-9]{1}[0-9]{1}$']
+                       [field: '$.update.release.dist_tag', expectedValue: '^(f[3-9]{1}[0-9]{1}|epel9|epel10)$']
```

Comment:
You can see the dist-tag for EPEL packages in Bodhi:
https://bodhi.fedoraproject.org/updates/?search=&releases=EPEL-10.0

Or you can check the Bodhi messages in Fedora datagrepper.